common/winobjc.versioning.common.targets

```diff
@@ -147,6 +150,40 @@
   </Target>
 
 
+    <!--
+    APPX Versioning target
+
+    This target will modify the Appx manifest that is generated during the compilation of an app.
+    The version of the app is modified as well as the display name to make it easy to identify sample app versions.
+  -->
+
+  <Target Name="FixUpAppxManifest"
+          BeforeTargets="_ValidateAppxManifest"
+          DependsOnTargets="_GenerateCurrentprojectAppxManifest"
```

Comment:
Where is _GenerateCurrentprojectAppxManifest defined? Also it's weird that project is lowercase.